app/static/joke.js

```diff
@@ -0,0 +1,5 @@
+function getDailyJoke() {
+  fetch('/getjoke')
+    .then(response => response.json())
+    .then(data => alert(data.value.joke));
```

Comment:
You can do much better :P

Reply:
I didn't really understand this one :)
I have change the logic with an event listener. I really like it popping. If it's ok.

Reply:
If you like the popup, please use [Bootstrap Models](https://getbootstrap.com/docs/5.0/components/modal/) instead - and not JS alert

Reply:
You can also use the JS library that I used for the text editor.
It's called sweetalert2, its in the text_editor.js

Reply:
I used SweetAlert2 as you suggested